### DISCUSSION TITLE
Query partial fields with `curl` when getting fzf status?

### DISCUSSION THREAD

**Discussion by linrongbin16** [junegunn/fzf#3611] [Q&A] [answered]
Hi,

The `fzf` has an experimental feature, that allows user get fzf status with `curl`.

Is there a way to only query specific fields that users are interesting? this is mostly for better performance. Since the status json payload is super large.

**Comment by junegunn** [comment, marked as answer]
Not at the moment. But you can pass `limit=0` to reduce the size.

```sh
curl -s localhost:6266?limit=0
```

**Reply by linrongbin16** [reply]
thank you!
